zookeeper-server/src/main/java/org/apache/zookeeper/Op.java

```diff
@@ -54,6 +56,35 @@
         READ
     }
 
+    /**
+     * Optional flags for the
+     * {@link Op#create(String, byte[], List, CreateMode, EnumSet)}
+     * and similar operations.
+     */
+    public enum CreateFlags {
+        /**
+         * Requests that a {@code Stat} object be included in the
+         * response.
+         *
+         * This is semantically equivalent to passing a non-{@code null}
+         * {@code stat} parameter to
+         * {@link ZooKeeper#create(String, byte[], List, CreateMode, Stat)}.
+         */
+        WITH_STAT;
```

Comment:
RETURN_STAT ?

Reply:
You suggested `WITH_STAT` or `RETURN_STAT`, so… I used both :)

More seriously, there is already a `RETURN_STAT` a few lines below.  `WITH_STAT` is the enumeration value, and `RETURN_STAT` the constant `Set`.

I'm not very happy with that.  But I suppose we actually don't need a set of flags if we use the builder approach; individual booleans are fine if the setters are named.